lib/minfraud/components/credit_card.rb

```diff
@@ -8,16 +8,18 @@ module Components
     class CreditCard < Base
       include Minfraud::Validates
 
-      # The issuer ID number for the credit card. This is the first 6 digits of
-      # the credit card number. It identifies the issuing bank.
+      # The issuer ID number for the credit card. This is the first 6 or 8
+      # digits of the credit card number. It identifies the issuing bank.
       #
       # @return [String, nil]
       attr_accessor :issuer_id_number
 
-      # The last four digits of the credit card number.
+      # The last digits of the credit card number. This is the last 4
+      # digits if the issuer ID number is 6 digits, or the last 2 digits
+      # if the issuer ID number is 8 digits.
```

Comment:
Hmm, this now seems contrary to what we say in the changelog :|. I guess we could only have the first sentence or something. Perhaps we could link somewhere for more info but I don't think we have anything yet, do we?

Reply:
I can add a link to https://dev.maxmind.com/minfraud/api-documentation/requests?lang=en#schema--request--credit-card__last_4_digits (but `s/last_4/last`) after https://github.com/maxmind/dev-site/pull/272 is merged